systemtest/src/test/java/io/strimzi/systemtest/upgrade/StrimziUpgradeST.java

```diff
@@ -91,8 +99,8 @@
     // ExpectedTopicCount contains additionally consumer-offset topic, my-topic and continuous-topic
     private final int expectedTopicCount = upgradeTopicCount + 3;
 
-    private final String latestReleasedOperator = "https://github.com/strimzi/strimzi-kafka-operator/releases/download/0.19.0/strimzi-0.19.0.zip";
-
+    private final String latestReleasedVersion = "0.19.0";
```

Comment:
Out of curiosity, what is the philosophy of this test with regards to versions? Does it always run operator upgrade against previous release without the version field? Or does it run Kafka upgrade from older Kafka version without upgrade?

For 0.19.0 it does both. But for example 0.18.0 -> 0.19.0 IIRC does not change Kafkas so it would test only operator upgrade. I think this should be clarified in the comments to know how and when should the version number be bumped. (in general, both tests might have their value)

Reply:
AFAIK this should test ability of upgrade the CO and then Kafka when the Kafka version is set to `null`. The CO should roll the pods after upgrade and then the version of Kafka should be set.

TBH I was thinking about adding test for upgrading Kafka by version (so upgrade from `2.4.0` to `2.5.0`, then upgrade the CO to master and upgrade to Kafka to the `2.6.0`). If it's good idea and it will make some sense, then I can add it.

That's MPOV.

Reply:
IIRC we wanted there upgrade CO from version X, kafka Y, to CO version Z and kafka Y + 1 at the end